src/server/controller/assessment/generateMetaCache/dependencyEvaluator/evalDependencies/member.ts

```diff
@@ -15,7 +15,8 @@ const _includesVariableCache = (variables: Array<VariableCache>, variable: Varia
         v.variableName === variable.variableName &&
```

Comment:
Changes in this file are required to support col level metaCache, otherwise we only had single entry in metaCache for given variable dependants

Reply:
Without this dependandants would have only 1 variable relation.
E.g. when clearing a cell selecting now, it would show only 1 variable instead of all relevant variables
Then it would clear only the first variable.

Adding col level should be ok, if col is undefined (optional) the comparison is ok because `undef = undef`.

Reply:
TODO: Test and verify calculations (e.g. change forestArea), validations and meta cache for val/calc